src/tdywy.c

```diff
@@ -725,7 +755,12 @@ PetscReal TDyWYPressureNorm(TDy tdy,Vec U) {
     ierr = DMPlexGetPointGlobal(dm,c,&gref,&junk); CHKERRQ(ierr);
     if(gref<0) continue;
     ierr = PetscSectionGetOffset(sec,c,&offset); CHKERRQ(ierr);
-    tdy->dirichlet(&(tdy->X[c*dim]),&p);
+    if (tdy->dirichlet){
+      tdy->dirichlet(&(tdy->X[c*dim]),&p);
+    }
+    if (tdy->ops->computedirichletvalue) {
```

Comment:
Is there a reason why this is in here twice? There are a few places where you are moving from XX to ops->XX where things are repeated. Are you thinking to just add in the new method and phase out the old?

Reply:
I'm hoping that we move to the `tdy->ops->XYZ()`. What do you think? If that is okay, I can make additional commits in this PR to swap out all old methods with new methods.

Reply:
I am ok with moving to ops, this is needed for the fortran interface correct?

Reply:
Yes, those changes are for the fortran interface. I will make the necessary changes in this PR.